src/docs/blog/typesafe_configuration/pre/secrets.kt

```diff
@@ -7,9 +7,10 @@ import java.nio.ByteBuffer
 
 val s3 = OkHttp()
 
-fun readFile(secretKey: String, bucketKey: String): ByteBuffer {
-    return s3(Request(GET, "https://mybucket.s3.amazonaws.com/$bucketKey")).body.payload
-}
+fun readFile(secretKey: String, bucketKey: String): ByteBuffer =
```

Comment:
`secretKey` is not being used - not sure if intended. ([Corresponding Docs](https://www.http4k.org/blog/typesafe_configuration/#4_security)